lifelines/plotting.py

```diff
@@ -441,6 +441,7 @@ def plot_lifetimes(
     sort_by_duration=True,
     event_observed_color="#A60628",
     event_censored_color="#348ABD",
+    label_durations=False,
```

Comment:
Thoughts on always using the index values if provided a pd.Series? By default, the index is 0, 1, 2, ... which is identical to current behaviour. I feel: if a user provided a pd.Series with non-trivial index values, they expect to see them.

Reply:
The only issue I see with always labelling with values from the index is that the labelling becomes cluttered if there are lot of durations to plot, e.g. with the waltons example in the doc string.

Reply:
Hm, that might be exception. Do you have a quick screenshot of how cluttered it could get? 

(Also, walton's data doesn't have an non-trivial index, right?)

Reply:
I see what you are saying, I've updated the function to only add the labels if durations is a pd.Series with non-default index (had to add the check before the durations sort since that rewrites the index leaving no way to tell if it was non-default).